deel-2-flexbox-layout/opdracht-4/styles/styles.css

```diff
@@ -1 +1,62 @@
-/*Schrijf hier jouw CSS*/
\ No newline at end of file
+/*Schrijf hier jouw CSS*/
+
+body{
+    display: flex;
+    flex-direction: row;
+    min-height: 100vh;
+}
+
+.outer-col-1{
+    display: flex;
+    flex-direction: row;
```

Comment:
Officieel gezien mocht de header niet groeien of krimpen. Dus horen er nog 2 properties schrik & grow hierbij met value van 0.